Tests/GRPCCodeGenTests/Internal/Translator/IDLToStructuredSwiftTranslatorSnippetBasedTests.swift

```diff
@@ -262,7 +335,7 @@ final class IDLToStructuredSwiftTranslatorSnippetBasedTests: XCTestCase {
 
   func testSameGeneratedNameServicesSameNamespaceError() throws {
     let serviceA = ServiceDescriptor(
-      documentation: "Documentation for AService",
+      documentation: "/// Documentation for AService\n",
```

Comment:
I don't think the service or method docs should be pre-formatted. We should only pre-format the leading trivia at the top of the proto file.

Reply:
They are also extracted from the `Location` objects associated with the methods and services, so they are pre-formatted. I remember we've decided to have all the docs / comments from the proto file as pre-formatted and only the ones we generate  as `.docs`

Reply:
I thought we decided that only the copyright header would be pre-formatted because those are the only docs we should copy verbatim. I should've picked this up in the PR which added it.

The other docs are formatted by `protoSourceComments()` which inserts the leading `///`, which isn't actually necessary as the renderer can (and should) do it instead. 

Either way we can just leave it as is for now, it's not particularly important.